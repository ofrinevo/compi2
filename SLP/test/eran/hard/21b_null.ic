
class MainClass { 

	static void main (string[] args) { 
		
		int[] a = null;

		Library.println("null[2] ");
		Library.printi(a[2]);
		Library.println("");
		Library.println("Should not print this or get here.");

	}

}
