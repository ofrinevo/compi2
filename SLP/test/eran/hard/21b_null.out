null[2] 
Runtime Error: Null pointer dereference!
